torch/ao/quantization/fx/_lower_to_native_backend.py

```diff
@@ -945,6 +949,24 @@ def _lower_getattr_tensor_metadta_op(model: QuantizedGraphModule):
             args[0] = n.args[0].args[0]
             n.args = tuple(args)
 
+def _lower_static_special_op(model: QuantizedGraphModule, qconfig_map: Dict[str, QConfigAny]):
```

Comment:
can we just add it to https://github.com/pytorch/pytorch/blob/master/torch/ao/quantization/fx/_lower_to_native_backend.py#L94?